gulpfile.js

```diff
@@ -30,6 +31,14 @@ require("events").EventEmitter.prototype._maxListeners = 100;
 ** GULP TASKS
 */
 
+gulp.task("rtl", function() {
+    var styleFolder = "./src/OrchardCore.Themes/TheAdmin/wwwroot/Styles";
```

Comment:
No. We can't have a specific reference to this css in this file.
Also why only this file and not all the other ones?

Reply:
That's why I asked above, I'm not gulp expert, so we need a better way for doing this